docs/cedarling/tutorials/kotlin.md

```
@@ -7,12 +7,303 @@ tags:
+// Get a specific log entry by ID
+val logEntry: String = adapter.getLogById(logIds.first())
+
+// Get logs by tag (e.g., "System")
+val systemLogs = adapter.getLogsByTag("System")
+```
+
+---
+
+## Defined API
+
+Defined APIs are listed [here](https://janssenproject.github.io/developer-docs/jans-cedarling/bindings/cedarling-java/io/jans/cedarling/binding/wrapper/CedarlingAdapter.html).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)